src/GraphQL/Http/HttpResponseStreamWriter.cs

```diff
@@ -239,14 +239,14 @@ public override Task FlushAsync()
             return FlushInternalAsync(flushEncoder: true);
         }
 
-        protected override void Dispose(bool disposing)
+        protected override async void Dispose(bool disposing)
```

Comment:
You never should use async voids..

Reply:
Ahh yeah, I should have removed this pull request as i created the other one which seems fine.